db/static.go

```diff
@@ -0,0 +1,30 @@
+package db
+
+import (
+	// sqlite driver
+	_ "github.com/mattn/go-sqlite3"
+
+	"github.com/jmoiron/sqlx"
+	"github.com/pkg/errors"
+)
+
+var db *sqlx.DB
+
+// InitDB initializes the database
+func InitDB() error {
+	var err error
+	//db = sqlx.Connect("mysql", "root:root@localhost/db")
```

Comment:
remove before merging

Reply:
I would keep it in since it is temp anyways